package.json

```diff
@@ -57,7 +57,7 @@
     "jest": "^22.1.4",
     "lint-staged": "^6.1.0",
     "prettier": "^1.10.2",
-    "react-dom": "^16.2.0",
+    "react-dom": "^16.3.2",
```

Comment:
This was unnecessary. Please ignore. Which reminds me: why do we need this? @jquense